Index: AStyle/build/gcc/Makefile
===================================================================
--- AStyle/build/gcc/Makefile	(revisione 398)
+++ AStyle/build/gcc/Makefile	(copia locale)
@@ -33,7 +33,7 @@
 objdir = obj
 ipath=$(prefix)/bin
 CBASEFLAGS = -W -Wall -fno-rtti -fno-exceptions
-JAVAINCS   = -I$(JAVA_HOME)/include
+#JAVAINCS   = -I$(JAVA_HOME)/include
 CXX = g++
 INSTALL=install -o $(USER) -g $(USER)
 
